src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -2,38 +2,77 @@ package dev.supergrecko.kllvm.core.typedefs
 
 import dev.supergrecko.kllvm.contracts.Disposable
 import dev.supergrecko.kllvm.contracts.Validatable
+import dev.supergrecko.kllvm.core.values.InstructionValue
 import org.bytedeco.llvm.LLVM.LLVMBuilderRef
 import org.bytedeco.llvm.global.LLVM
 
 public class Builder internal constructor(internal val llvmBuilder: LLVMBuilderRef) : AutoCloseable, Validatable, Disposable {
     public override var valid: Boolean = true
 
-    override fun dispose() {
-        require(valid) { "This builder has already been disposed." }
+    //region Core::Builders
```

Comment:
These APIs come from the InstructionBuilders module so this should be named `//region InstructionBuilders`

Reply:
Fixed